src/components/details.js

```diff
@@ -0,0 +1,77 @@
+import React, { useEffect, useState } from 'react';
+import { useParams, useNavigate } from 'react-router-dom';
+import { useSelector } from 'react-redux';
+
+const Details = () => {
+  const { id } = useParams();
+  const push = useNavigate();
+  const animals = useSelector((state) => state.animals);
+  const [animal, setAnimal] = useState({});
+  useEffect(() => {
+    animals.forEach((an) => {
+      if (parseInt(an.id, 10) === parseInt(id, 10)) {
+        setAnimal(an);
+      }
+    });
+  }, []);
+  return (
+    <div className="Details">
+      <button type="button" onClick={() => push('/')}>
+        <i className="fa fa-arrow-left" aria-hidden="true" />
+        Home
+      </button>
+      <div className="animalDetails">
+        <h2>{animal.name}</h2>
+        <img src={animal.image} alt={`${animal.name} details`} />
+
+        <h2 className="characteristics">Characteristics</h2>
+        <p>
+          Latin Name
+          <span className="right">{animal.latinName}</span>
+        </p>
+        <p>
+          Animal Type
+          <span className="right">{animal.animalType}</span>
+        </p>
+        <p>
+          Active Times
+          <span className="right">{animal.activeTime}</span>
+        </p>
+        <p>
+          Minimum Length
+          <span className="right">{animal.minLength}</span>
+        </p>
+        <p>
+          Maximum Length
+          <span className="right">{animal.maxLength}</span>
+        </p>
+        <p>
+          Minimum Weight
+          <span className="right">{animal.minWeight}</span>
+        </p>
+        <p>
+          Maximum Weight
+          <span className="right">{animal.maxWeight}</span>
+        </p>
+        <p>
+          Life span
+          <span className="right">{animal.lifeSpan}</span>
+        </p>
+        <p>
+          Habitats
+          <span className="right">{animal.habitats}</span>
+        </p>
+        <p>
+          Diet
+          <span className="right">{animal.diet}</span>
+        </p>
+        <p>
+          Locations
+          <span className="right">{animal.locattions}</span>
+        </p>
+      </div>
```

Comment:
- [ ] Let's try to follow the design as much as possible. On your Details page, kindly add the stripe color as we have in the template.

- [ ] Ensure you maintain the appropriate spacing between elements.

  Your Implementation | Behance Design
  :---:|:---:
  ![nemwel-details](https://user-images.githubusercontent.com/36542986/164278614-97edebb9-32f8-4d5b-a3dd-d3cd6ee2743f.jpg) | ![jonathas-origin-detail](https://user-images.githubusercontent.com/36542986/164278607-101d01d3-0937-480d-8629-6e5fd891e678.jpg)

Reply:
Hello @TedLivist Thank you I have effected the change.